resources/us_autocompletions/models/us_autocompletions.yml

```diff
@@ -0,0 +1,35 @@
+type: object
+
+required:
+  - address_prefix
+
+properties:
+  address_prefix:
+    type: string
+    description: >
+      Only accepts numbers and street names in an alphanumeric format.
+
+  city:
+    type: string
+    description: >
+      An optional city input used to filter suggestions. Case insensitive, and
+      does not match partial abbreviations.
+
+  state:
+    type: string
+    description: >
+      An optional state input used to filter suggestions. Case insensitive, and does not match
+      partial abbreviations.
+
+  zip_code:
+    type: string
+    description: >
+      An optional ZIP Code input used to filter suggestions. Matches partial entries.
+
+  geo_ip_sort:
+    type: string
+    description: >
+      If true, sort suggestions by proximity to the IP set in the `X-Forwarded-For` header.
+
+  object:
```

Comment:
I'm sorry, I'm not very with it today. I actually saw this issue first time around but then lost track of it. Vaccinated brain is not happy brain.

The `object` property isn't a payload parameter. I'm not entirely sure why we provide it, but that's another story. I looked, and we made the same error in `us_verifications`. Please remove `object` from those payload schemas as well.